### ISSUE TITLE
About the last layer of the model

### ISSUE THREAD

**Issue by thuBingo** [miaow1988/ShuffleNet_V2_pytorch_caffe#29]
There is a conv layer at the last instead of a full connected layer. There isn't a dropout layer, I think it may lead to overfitting during training.
Actually I used the 1.0 model to train my dataset, and overfitting happens. (train acc 87 val acc 82)

**Comment by miaow1988** [comment]
Thanks. I'll try that.
